directory/tests/test_directory.py

```diff
@@ -288,6 +288,30 @@ def test_file_metrics_many(aggregator):
         assert aggregator.metrics_asserted_pct == 100.0
 
 
+def test_folder_metrics(aggregator):
+    config_stubs = common.get_config_stubs(temp_dir + "/main", collect_folder_stats=True)
+
+    for config in config_stubs:
+        aggregator.reset()
+        dir_check = DirectoryCheck('directory', {}, [config])
+        dir_check.check(config)
+        dirtagname = config.get('dirtagname', "name")
+        name = config.get('name', temp_dir + "/main")
+        dir_tags = [dirtagname + ":%s" % name, 'optional:tag1']
+
+        # File metrics
```

Comment:
Not gonna lie, it's a bit confusing when the comment says "file metrics" but the variable says "folder metrics", and the following block says "common" vs "directory" metrics ;)

Suggestion: remove the inline comments, they're redundant at best.